app/assets/javascripts/deploy_groups.js

```diff
@@ -0,0 +1,17 @@
+// get back a list of deploy groups, generate a select list with 'em
+// these are being stored in the DOM to avoid having to make an ajax query etc
+
```

Comment:
this is only used on 1 page, right ?
how about adding js to the page to match the select on-ready ?